core/tauri/src/image/mod.rs

```diff
@@ -186,20 +188,39 @@ impl TryFrom<Image<'_>> for tray_icon::Icon {
   }
 }
 
+/// The counterpart of the `@tauri-apps/api/image` Image class.
+///
+/// # Stability
+///
+/// The stability of the variants are not guaranteed, and matching against them is not recommended.
+/// Use [`JsImage::into_img`] instead.
 #[derive(serde::Deserialize)]
 #[serde(untagged)]
+#[non_exhaustive]
 pub enum JsImage<'a> {
```

Comment:
`Image` struct is re-exported from root, do we want to do the same for `JsImage` or remove the re-export from root? or just keep different re-exports?

Reply:
i feel like we have too many root re-exports tbh